### ISSUE TITLE
Add support for QBASIC dialect/IDE Features

### ISSUE THREAD

**Issue by wkmanire** [dykstrom/basic-mode#9]
Would you be willing accept contributions that add support of the QBASIC dialect of BASIC? There is a list of additional keywords that need highlighting. Also, there would need to be modifications for indentation of FUNCTION and SELECT CASE.

The QBASIC IDE automatically uppercases keywords as that was the convention for the dialect. There are probably a couple of other minor things from the IDE that could be brought into this mode as well.

If you are receptive to the idea, I can break this request down into a number of smaller issues and also commit to making contributions where possible.

**Comment by dykstrom** [comment]
I am happy for all contributions that bring value to the basic-mode package. My original idea with basic-mode was to support the QuickBasic dialect (which I think is the same as QBASIC) while at the same time try to support as much as possible of other BASIC dialects as well. Everything has not been implemented yet, and you are very welcome to contribute.

However, I would like to have some things configurable, for example upcase or downcase for keywords. I  believe there are also BASIC dialects using title case for keywords. My idea was that in the future, basic-format-code should modify the case of keywords, functions etc, and the user would use customize to specify in which way.

So in summary, you are welcome to contribute, but please make things that may clash with other BASIC dialects configurable if possible.

**Comment by wkmanire** [comment]
Great! I'll begin cutting issues for the QBasic dialect.
